### ISSUE TITLE
File not found - *.gt.txt

### ISSUE THREAD

**Issue by jimlaloi** [tesseract-ocr/tesstrain#369]
I'm new to tesstrain, following the readme to get started. I'm getting an error that doesn't make any sense to me. I'm on Windows 10. Here's what I've done:

- Installed Tesseract (I have version 5.3.3.20231005) and added it to PATH
- Installed Git SCM (linked in the readme) and added C:\Program Files\Git\usr\bin to PATH
- Installed Windows Package Manager (linked in the readme) ran `winget install GnuWin32.Make` and `winget install wget`
- When trying to run Make after this, it said it was outdated, so I installed a newer version with `choco install make`. After this Make appeared to be working. I have Make version 4.4.1.
- Installed Bc and dc calculator and unzipped bc.exe to my PATH.
- I cloned the tesstrain repository, created a subdirectory data/foo-ground-truth, and unzipped the contents of ocrd-testset.zip into it.
- I ran `make tesseract-langdata`, which successfully downloaded a bunch of unicharset files into data/langdata

Now for the error. When I try to run `make training`, I get the following error:
```
File not found - *.gt.txt
File not found - *.gt.txt
    You are using make version: 4.4.1
Makefile:224: *** found no data/foo-ground-truth/*.gt.txt for data/foo/all-gt.  Stop.
```
I have confirmed that I do indeed have a whole bunch of .gt.txt files in data/foo-ground-truth, from the unzipped ocrd-testset.zip. I've hit a wall on my troubleshooting. What might be causing it to not find these files?

**Comment by stweil** [comment]
`find` in a Windows command line is not the same as the POSIX `find`, for example in a Linux shell. Training won't work in a Windows command line. I suggest to use WSL on Windows.

**Comment by zdenop** [comment]
WSL is a good suggestion, but POSIX find is part of the git installation - make sure that the git path(s) are at the beginning of PATH env variable.

**Comment by jimlaloi** [comment]
@zdenop Thanks you pointed me in the right direction. I double-checked my PATH env variable order from the command line (`path`), and the git path was halfway down. I wasn't sure why, because I had put it at the top of my PATH env variable in the Adv System Settings dialog. But then I realized I had put it in my User PATH, which is listed after all the paths in the System PATH env variable. Once I added my git path to the System PATH (at the top), everything started working.
